backend.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+    backend "s3" {
+        bucket = "sureshdemo22"
+        key = "devopstfdemo/backenddemo"
+        region = "ap-south-1"
+        dynamodb_table = "tfdemo"
+      

```

Comment:
Code formate is not as per our standard

Reply:
Changes has been made, please review again